src/roadmap/v1/lifecycle/systems.py

```diff
@@ -14,14 +14,14 @@
 async def get_systems():
     systems = get_systems_data()
 
-    return sorted(systems, key=lambda d: (d["major"], d["minor"]))
+    return sorted(systems, key=lambda d: (d["major"], d["minor"]), reverse=True)
```

Comment:
I generally try to avoid using `lamdba` in Python since they're pretty unreadable. I found a really nice way of doing this the other day using the [`operator` module](https://docs.python.org/3/library/operator.html#operator.attrgetter) and it does exactly what we need here.

> Return a callable object that fetches attr from its operand. If more than one attribute is requested, returns a tuple of attributes.

```suggestion
    from operator import attrgetter
    return sorted(systems, key=attrgetter("major", "minor"), reverse=True)
```